docs/src/main/sphinx/connector/delta-lake.rst

```diff
@@ -159,6 +163,9 @@ values. Typical usage does not require you to configure them.
     * - ``delta.register-table-procedure.enabled``
       - Enable to allow users to call the ``register_table`` procedure.
       - ``false``
+    * - ``delta.vacuum.min-retention``
+      - Minimal retention period for the :ref:`VACUUM<delta-lake-vacuum>` procedure.
```

Comment:
what is this actually ... the minimum duration (link to prop type) or age of files of ?? .. younger files are removed by vacuum?

Reply:
Added more information to avoid misleading the users about this configuration property.